src/Appearance.jsx

```diff
@@ -1,9 +1,432 @@
+                src="https://mfe-appearance.production.linktr.ee/images/selector-olivia-rodrigo-guts.bce5704bb397972aa4c6.webp"
+              />
+            </Flex>
+            <Text w="128px" textAlign="center">
+              Linktree x Olivia Rodrigo
+            </Text>
+          </Flex>
+          <Flex flexDir="column" alignItems="center">
+            <Flex h="200px" w="128px" mb="10px" _hover={{ cursor: "pointer" }}>
+              <Image
+                borderRadius="10px"
+                src="https://mfe-appearance.production.linktr.ee/images/selector-olivia-rodrigo-guts-v2.48846b855e65ab113f62.webp"
+              />
+            </Flex>
+            <Text w="128px" textAlign="center">
+              Linktree x Olivia Rodrigo (Dark)
+            </Text>
+          </Flex>
+          <Flex flexDir="column" alignItems="center">
+            <Flex h="200px" w="128px" mb="10px" _hover={{ cursor: "pointer" }}>
+              <Image
+                borderRadius="10px"
+                src="https://mfe-appearance.production.linktr.ee/images/selector-pebble-blue.48047a34097e85836255.png"
+              />
+            </Flex>
+            <Text w="128px" textAlign="center">
+              Pebble Blue
+            </Text>
+          </Flex>
+        </Flex>
+      </Flex>
+      <Text mt="48px" fontSize="1.4rem">
+        Custom Appearance
+      </Text>
+      <Text mt="18px" fontSize="1rem" maxW="600px">
+        Completely customize your Linktree profile. Change your background with
+        colors, gradients and images. Choose a button style, change the typeface
+        and more.
+      </Text>
+      <Text mt="48px" mb="16px" fontSize="1.4rem">
+        Backgrounds
+      </Text>
+      <Flex flexDir="column" bg="gray.700" borderRadius="30px">
+        <Flex my="28px" mx="16px" justifyContent="space-evenly">
+          <Flex flexDir="column" alignItems="center">
+            <Flex
+              h="200px"
+              w="128px"
+              mb="10px"
+              borderRadius="10px"
+              _hover={{ cursor: "pointer" }}
+              bg="gray.400"
+            ></Flex>
+            <Text>Flat Color</Text>
+          </Flex>
+          <Flex flexDir="column" alignItems="center">
+            <Flex
+              h="200px"
+              w="128px"
+              mb="10px"
+              borderRadius="10px"
+              border="solid"
+              _hover={{ cursor: "pointer" }}
+            ></Flex>
+            <Text w="128px" textAlign="center">
+              Gradient
+            </Text>
+          </Flex>
+          <Flex flexDir="column" alignItems="center">
+            <Flex
+              h="200px"
+              w="128px"
+              mb="10px"
+              borderRadius="10px"
+              border="dotted"
+              _hover={{ cursor: "pointer" }}
+            ></Flex>
+            <Text w="128px" textAlign="center">
+              Image
+            </Text>
+          </Flex>
```

Comment:
This needs to be refactored to use one reusable component. Then pass in the dynamic data through props. Ex:
`<ThemePreview label="Pebble Blue" url="asdf.com" />`